### PR TITLE
Fixed the attendance lists

### PR DESCRIPTION
**Pull request by TT94ka**

### CODE REVIEW

File: api/LMPlatform.UI/Controllers/SubjectController.cs [hunk 1]
```diff
@@ -416,7 +419,24 @@ public DataTablesResult<SubjectListViewModel> GetSubjects(DataTablesParam dataTa
                 subjects.TotalCount);
         }
 
-        
+        [HttpGet]
+        public ActionResult GetSubjectModulesForSchedule(int subjectId)
+        {
+
```
Comment (hawkrai) [inline api/LMPlatform.UI/Controllers/SubjectController.cs]: empty lines

File: api/LMPlatform.UI/Controllers/SubjectController.cs [hunk 2]
```diff
@@ -416,7 +419,24 @@ public DataTablesResult<SubjectListViewModel> GetSubjects(DataTablesParam dataTa
...
+            var modules = ModulesManagementService.GetModules(subjectId);
+            var subjectModules = new
+            {
+                hasLecture = modules.Where(e => e.ModuleType == ModuleType.Lectures).Any(),
```
Comment (hawkrai) [inline api/LMPlatform.UI/Controllers/SubjectController.cs]: a very inefficient code, needs to be rewritten to a more efficient one

**Review by hawkrai — CHANGES_REQUESTED** [review]

File: api/LMPlatform.UI/Services/Practicals/PracticalService.svc.cs [hunk 3]
```diff
@@ -354,24 +354,27 @@ public PracticalsResult GetPracticalsV2(int subjectId, int groupId)
                         Mark = string.Empty
                     })?.ToList()
                 }).ToList();
-                var durationCount = 0;
 
+                int durationCount = 0;
```
Comment (hawkrai) [inline api/LMPlatform.UI/Services/Practicals/PracticalService.svc.cs]: Why convert a var into a specific type?

Comment (TT94ka) [reply api/LMPlatform.UI/Services/Practicals/PracticalService.svc.cs]: This makes the code more understandable and accessible.

Comment (hawkrai) [reply api/LMPlatform.UI/Services/Practicals/PracticalService.svc.cs]: Read basic practices and approaches.
The value determines the type. it makes it clear what type of data will be there.

**Pull request merged**